docs/source/bettertransformer/tutorials/content.mdx

```diff
@@ -0,0 +1,83 @@
+<!--Copyright 2022 The HuggingFace Team. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+the License. You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+-->
+# How to use `optimum` and `BetterTransformer`?
+
+## Install dependencies
+
+You can easily une the `BetterTransformer` API using HuggingFace `optimum`, first install the dependencies as follows:
+
+```bash
+pip install transformers accelerate optimum
+```
+
+Also, make sure to install the latest version of PyTorch by following the guidelines on the [PyTorch official website](https://pytorch.org/get-started/locally/). Note that `BetterTransformer` API is only compatible with `torch>=1.13`, so make sure to have this version installed on your environement before starting.
+
+## Step 1: Load yout HuggingFace model
+
+First, load your HuggingFace model using `transformers`. Make sure to download one of the models that is supported by the `BetterTransformer` API:
+
+```python
+>>> from transformers import AutoModel
+
+>>> model_id = "roberta-base"
+>>> model = AutoModel.from_pretrained(model_id)
+```
+(Sometimes you can directly load your model on your GPU devices using `accelerate` library, therefore you can optionally try out the following command:)
+
+```python
+>>> from transformers import AutoModel
+
+>>> model_id = "roberta-base"
+>>> model = AutoModel.from_pretrained(model_id, device_map="auto")
+```
+
+## Step 2: Set your model on your preferred device
```

Comment:
What's the difference with device_map=auto? Couldn't we move the Accelerate part at the end of the tutorial since it is optional?